mevislab.github.io/content/tutorials/basicmechanisms/macromodules/pythonpip.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Example 4: Install additional Python packages via PythonPip module"
+date: 2023-05-16
+status: "OK"
+draft: false
+weight: 450
+tags: ["Advanced", "Tutorial", "Python", "PythonPip", "pip"]
+menu: 
+  main:
+    identifier: "pythonpip"
+    title: "Install additional Python packages via PythonPip module"
+    weight: 450
+    parent: "basicmechanisms"
+---
+# Example 4: Install additional Python packages via PythonPip module
+## Introduction
+MeVisLab already comes with a lot of integrated third party software tools ready to use. Nevertheless it might be necessary to install additional Python packages for your specific needs. This example shows the usage of the `PythonPip` module in MeVisLab for installing additional Python packages.
+
+The `PythonPip` module allows to work with the Python package manager pip. It can be used to install Python packages into the site-packages of the MeVisLab Python installation.
+
+It provides the full Python package eco-system to MeVisLab, but it has some drawbacks:
+
+* packages can contain C-Extensions (since we use the same MSVC compiler resp. same GCC settings as Python 3 itself)
+* *but* you can only install packages that do not interfere with packages or DLLs that are already part of MeVisLab
+* this means that installing packages with C-Extensions might work in many circumstances, but is not guaranteed to work
+* **all installed packages with C-Extensions are release only**, so you can only import them in a release MeVisLab (under Windows)
```

Comment:
I find it a bit confusing that there are 4 bullet point for two drawbacks - the first 3 lines belong together